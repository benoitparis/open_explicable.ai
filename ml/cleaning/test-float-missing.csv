col_num_float,col_num_float_no_data
1.0,1.0
2.0,2.0
2.0,