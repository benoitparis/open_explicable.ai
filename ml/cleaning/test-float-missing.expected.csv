col_num_float,col_num_float_no_data,col_num_float_no_data_missing
1.0,1.0,0
2.0,2.0,0
2.0,-1.0,1
